openslides/users/views.py

```diff
@@ -159,12 +159,17 @@ def reset_password(self, request, pk=None):
         """
         View to reset the password using the requested password.
         """
-        user = self.get_object()
-        if request.data.get('password'):
-            user.default_password = request.data['password']
-        user.set_password(user.default_password)
-        user.save()
-        return Response({'detail': _('Password successfully reset.')})
+        # check manager perms
+        if request.user.has_perm('users.can_manage'):
+            user = self.get_object()
+            if isinstance(request.data.get('password'), str):
+                user.set_password(request.data.get('password'))
```

Comment:
Let me explain our idea (from Finn and me): 
* The initial password is set onetime with creation of a new user. Then this initial password is fixed and not changeable.
* The access PDF prints always the initial password. This PDF is always valid. It's not possible that the initial password changed and so the PDF becomes invalid.
* The admin can change the personal password of an user only - like the user can do it in its own password change view. There are two use cases how a admin can do it:
 * The user stands near the admin PC and the user enter his new password.
 * The admin reset the current password to the initial password (by coping of the printed initial password) and says to user: You can reuse the already printed initial password from PDF.

Our idea is to reduce the confusion of resting the user password. There should be only ONE password which can changed. The initial password is for onetime login and printing on PDF.

Hope this is clear now. Otherwise ask me personally.